extensions/mongodb-client/deployment/src/test/java/io/quarkus/mongodb/NamedMongoClientConfigTest.java

```diff
@@ -0,0 +1,42 @@
+package io.quarkus.mongodb;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import javax.inject.Inject;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+
+public class NamedMongoClientConfigTest extends MongoWithReplicasTestBase {
+
+    @Inject
+    @io.quarkus.mongodb.runtime.MongoClient("cluster1")
+    ReactiveMongoClient client;
+
+    @Inject
+    @io.quarkus.mongodb.runtime.MongoClient("cluster2")
+    ReactiveMongoClient client2;
+
+    @AfterEach
+    void cleanup() {
+        if (client != null) {
+            client.close();
+        }
+        if (client2 != null) {
+            client2.close();
+        }
+    }
+
+    @RegisterExtension
```

Comment:
IIRC, we usually place this at the top of the class, so best keep things consistent

Reply:
ok